frontend/appflowy_flutter/lib/user/application/sign_in_bloc.dart

```diff
@@ -69,6 +69,11 @@ class SignInBloc extends Bloc<SignInEvent, SignInState> {
               ),
             );
           },
+          switchLoginType: (type) {
+            emit(
+              state.copyWith(loginType: type),
+            );
+          },
```

Comment:
```suggestion
          toggleLoginType: () => emit(
              state.copyWith(
                loginType: state.loginType == LoginType.signIn
                    ? LoginType.signUp
                    : LoginType.signIn,
                  ),
            ),
```
_personal preference_